hasona23.ShiftsLogger/ShiftsLogger/Program.cs

```diff
@@ -0,0 +1,33 @@
+using Microsoft.EntityFrameworkCore;
+using ShiftsLoggerAPI;
+using ShiftsLoggerAPI.Services;
+using Microsoft.OpenApi.Models;
+
+var builder = WebApplication.CreateBuilder(args);
+
+builder.Services.AddControllers();
+
+// Configure Entity Framework with your DbContext
+builder.Services.AddDbContext<ShiftsLoggerContext>(options =>
+    options.UseSqlServer(builder.Configuration.GetConnectionString("DefaultConnection")));
```

Comment:
I am confused. It looks like you are reading a Connection String from the configuration file, but they ignoring it and using a hard-coded value in the DBContext implementation.